src/main/scala/cromwell/engine/workflow/WorkflowManagerActor.scala

```diff
@@ -137,8 +137,9 @@ class WorkflowManagerActor(backend: Backend) extends Actor with CromwellActor {
     for {
       _ <- assertWorkflowExistence(id)
       outputs <- globalDataAccess.getOutputs(id)
+      filteredOutputs = outputs filter { _.isWorkflowOutput }
```

Comment:
If we're going to write the workflow outputs field to the symbol table I think this should be filtered in the DB and not in memory